android/src/main/java/com/rcttabview/RCTTabView.kt

```diff
@@ -25,6 +28,7 @@ class ReactBottomNavigationView(context: Context) : BottomNavigationView(context
   var items: MutableList<TabInfo>? = null
   var onTabSelectedListener: ((WritableMap) -> Unit)? = null
   private var isAnimating = false
+  private var barTintColor : Int? = null
```

Comment:
Why do we need to store this? We can pass the value from setBarTintColor directly to `updateBackgroundColors` method.

Reply:
Yeah this makes sense Changing this